src/magento-payments/voids.md

```diff
@@ -1,3 +1,16 @@
 ---
 title: Voids
----
\ No newline at end of file
+---
+
+With Magento Payments, you can utilize existing Magento functionality for voiding transactions. Voids allow you to stop payment on a specific order, if the funds are not yet captured. After you void an order, you can cancel orders in `Pending` status and issue any needed refunds.
```

Comment:
Definitely need Craig's $0.02 here but I think it might be more correct to say "Voids allow you to release a hold on funds for a specific order".

Reply:
Voids allow you to stop payment on a specific order, if the funds are not yet captured. After you void an order, you can cancel orders in Pending status and issue any needed refunds.
to
Voids allow you to liberate funds blocked by authorisation for an order as long as it has not yet been captured. After you void an authorisation, you can cancel the order in Pending status. Alternatively, you could just choose to cancel an order and any uncaptured authorisations will also be voided.

(extra note of explanation - refunds are only possible/necessary on captured transactions).